server/src/test/java/org/elasticsearch/cluster/coordination/stateless/HeartbeatTestSerializationTests.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.cluster.coordination.stateless;
+
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.test.AbstractWireSerializingTestCase;
+
+import java.io.IOException;
+
+public class HeartbeatTestSerializationTests extends AbstractWireSerializingTestCase<Heartbeat> {
```

Comment:
naming nit/suggestion:

```suggestion
public class HeartbeatSerializationTests extends AbstractWireSerializingTestCase<Heartbeat> {
```